test/acceptance/pages.py

```diff
@@ -15,6 +15,34 @@ class PageConfigurationError(Exception):
     pass
 
 
+class MultipleAssessmentPage(PageObject):
```

Comment:
BaseAssessmentPage

Remove is_browser_on_page

Have MultipleAssessmentPage extend BaseAssessmentPage and add is_browser_on_page.

Also have OpenAssessmentPage extend BaseAssessmentPage.